src/uu/wc/src/wc.rs

```diff
@@ -364,24 +369,54 @@ fn print_stats(
         min_width = 0;
     }
 
+    let mut is_first: bool = true;
+
     if settings.show_lines {
-        write!(stdout_lock, "{:1$}", result.count.lines, min_width)?;
+        if is_first {
+            write!(stdout_lock, "{:1$}", result.count.lines, min_width)?;
```

Comment:
this is a lot of duplicate declarations
what about doing the string declaration (ex: "{:1$}") in the "if"
and do the write declaration online once?

Reply:
this could help a little:
```rust
if !is_first {                                               
    write!(stdout_lock, " ")?;                               
}                                                            
write!(stdout_lock, "{:1$}", result.count.lines, min_width)?;
is_first = false;                                            
```

Reply:
Yes, that's a bit better, I'll make that change.